makefile

```diff
@@ -1,19 +1,80 @@
-format:
```

Comment:
optional question: Are there big differences between using a makefile vs. setuptools for command/build management?

Reply:
I wasnt aware we could use setuptools for this, so I dont really know